composer.json

```diff
@@ -68,5 +69,6 @@
             "assets:install --symlink --relative %PUBLIC_DIR%": "symfony-cmd",
             "assets:install %PUBLIC_DIR%": "symfony-cmd"
         }
-    }
+    },
+    "minimum-stability": "RC"
```

Comment:
I would also adds `prefer-stable: true`

Reply:
I ran composer update with that command option.